src/java.base/share/classes/sun/net/util/IPAddressUtil.java

```diff
@@ -134,22 +134,39 @@ public static byte[] textToNumericFormatV4(String src)
      * If string can't be parsed by following IETF IPv4 address string literals
      * formatting style rules (default one), but can be parsed by following BSD formatting
```

Comment:
I realize that this is an existing javadoc comment in an internal utility class (which is used by public InetAddress API implementation); do you think we should add a more formal reference to the BSD formatting of IP address, like we currently do for other RFCs and such in the class level javadoc of `InetAddress`?